src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -28,12 +36,23 @@ export const MyFieldAndBackground = () => {
 
   const person = personQuery.data;
   const personBackground = person?.background ?? {};
+  const personKeywords = person?.keywords ?? [];
+
+  const fieldQuery = useQuery({
+    enabled: !!debouncedSearchTerm,
+    queryKey: ['field', debouncedSearchTerm],
+    queryFn: () => searchForKeywords(25, 1, debouncedSearchTerm),
+    meta: { message: 'error', variant: 'error' },
+  });
+
+  const keywordsResult = fieldQuery.data?.hits ?? [];
 
   const initialValues: PersonBackgroundFormData = {
     background: {
       no: personBackground.no ? personBackground.no : '',
       en: personBackground.en ? personBackground.en : '',
     },
+    keywords: personKeywords ? personKeywords : [],
```

Comment:
går for det samme, men ser kanskje litt ryddigere ut? samme gjelder vel for background over også
```suggestion
    keywords: personKeywords ?? [],
```